cpg_workflows/large_cohort/sample_qc.py

```diff
@@ -8,20 +8,22 @@
 import hail as hl
 from cpg_utils import Path
 from cpg_utils.config import get_config
-from cpg_utils.hail_batch import genome_build, reference_path
+from cpg_utils.hail_batch import genome_build, image_path, query_command, reference_path
 from gnomad.sample_qc.pipeline import annotate_sex
+from hailtop.batch import Batch
+from hailtop.batch.job import Job
 
 from cpg_workflows.inputs import get_cohort
 from cpg_workflows.utils import can_reuse
 
 
 def run(
-    vds_path: Path,
-    out_sample_qc_ht_path: Path,
-    tmp_prefix: Path,
+    vds_path,
+    out_sample_qc_ht_path,
+    tmp_prefix,
 ):
```

Comment:
can you workshop this a bit? The type hinting here should identify these are `str` types, which means that you can remove a couple of `str(X)` transformations of these variables below, but `tmp_prefix` needs to be run through `to_path()` for this kind of syntax to work: `sqc_ht_path = tmp_prefix / 'sample_qc.ht'`